src/resolvers-cassandra/Edges.js

```diff
@@ -10,12 +10,26 @@ module.exports = {
   },
 
   saveLocations(args, res){ // eslint-disable-line no-unused-vars
+    return new Promise((resolve, reject) => { // eslint-disable-line no-unused-vars
+      reject(
+        'This API call is no longer supported. ' +
+        'We now automatically filter events down to only those ' +
+        'locations defined in the geo-fence for your site'
+      );
+    });
```

Comment:
I was wondering that as well. Also, does saveLocations in /resolvers need to be changed, too?